test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -1164,6 +1164,25 @@ def matcher_check_fn():
             quantization_with_autocast=quantization_with_autocast,
         )
 
+        if torch._inductor.config.cpp_wrapper:
+            self._test_code_common(
+                mod,
+                (v,),
+                [f"aoti_torch_{device}__qconv_pointwise_tensor"],
```

Comment:
Will the `tensor` variant always be used here?

Reply:
Yes, we now always use qconv_pointwise.tensor for both int8 and fp8.